microshift_cli_ref/microshift-usage-oc-kubectl.adoc

```diff
@@ -59,4 +59,4 @@ The `kubectl` binary is provided as a means to support existing workflows and sc
 
 The `kubectl` binary is included in the archive if you download the `oc` binary.
 
-For more information, see the link:https://kubernetes.io/docs/reference/kubectl/overview/[kubectl documentation].
+For more about `kubectl`, read the link:https://kubernetes.io/docs/reference/kubectl/overview/[Kubernetes CLI tool documentation].
```

Comment:
In the preview after this line i do see a text which does not match with the above content. Can we remove that ?

Reply:
@kasturinarra That's a footnote for the table in the oc binary section. The asciidoc there is an issue, so I moved the sections around to align the footnote directly with the table it references.

Reply:
Yes, you see the updated preview here: https://60414--docspreview.netlify.app/microshift/latest/microshift_cli_ref/microshift-usage-oc-kubectl.html
It's not pretty, but now the footnote makes sense.